deployments/multiple-ragapps/data/traefik/dynamic.yml

```diff
@@ -0,0 +1,20 @@
+http:
+  middlewares:
+    redirect-to-https:
+      redirectScheme:
+        scheme: https
+        permanent: true
+
+  routers:
+    # Redirect all HTTP traffic to HTTPS (except localhost)
```

Comment:
can we also redirect `www.${DOMAIN}` to `${DOMAIN}`?

Reply:
I considered that, but adding a www redirect complicates the config. Instead, set it in DNS setting of the domain is much easier. We can add note in README about that, the users then can set DOMAIN=www.domain.com if they prefer to use `www`

Reply:
sounds good, let's keep it simple then